src/main/java/core/basesyntax/models/RightTriangle.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.models;
+
+import core.basesyntax.Figure;
+
+public class RightTriangle implements Figure {
+    private final String color;
+    private final double firstLeg;
+    private final double secondLeg;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        this.color = color;
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Color: " + color
+                + "Type: Right Triangle "
+                + "First Leg: " + firstLeg
+                + "Second Leg: " + secondLeg);

```

Comment:
It is also recommended to add a space before 'Second Leg:' for consistency and readability in the output.